packages/components/src/twin.d.ts

```diff
@@ -0,0 +1,28 @@
+import "twin.macro";
```

Comment:
followed https://github.com/ben-rogerson/twin.examples/tree/master/react-styled-components#complete-the-typescript-setup

Reply:
🤔 Interesting. Would be nice if tw shipped with type definitions for its different integrations.